src/components/DonationPageComponents/DonateWindow.js

```diff
@@ -0,0 +1,225 @@
+            .substr(0, 16);
+        const parts = [];
+
+        // Split into 4 character chunks
+        for (let i = 0; i < v.length; i += 4) {
+            parts.push(v.substr(i, 4));
+        }
+
+        return parts.length > 1 ? parts.join(" ") : value;
+    }
+
+    function cc_format_expdate(value){
+        const v = value
+        // Remove all non-digit characters
+            .replace(/\s+/g, "")
+            .replace(/\D/g, "")
+            .substr(0, 4);
+        const parts = [];
+
+        // Split into 2 character chunks
+        for (let i = 0; i < v.length; i += 2) {
+            parts.push(v.substr(i, 2));
+        }
+
+        // THis 
+        return parts.length > 1 ? parts.join(" / ") : value;
+    }
+
+
+  return (
+    
+    <div className='bg-gray-100 p-4'>
+        <div className='text-2xl font-bold '>
+            You are supporting {props.title}
+            <div className='text-sm italic font-normal'>You have until {props.plannedEndDate} to support the cause</div>
+        </div>
+        <div className='flex justify-center text-3xl pt-6'>
+            Help make a difference
+        </div>
+        <form onSubmit={handleSubmit} >
+            <div className='flex  p-4 justify-center border-b'>
+                <div className='flex flex-col relative'>
+                    <label className='flex justify-center'>
+                        <div className='absolute top-4 left-3 font-bold text-3xl'>$</div>
+                            <input
+                                className='border-2 border-gray-300 p-4 rounded-lg text-right text-3xl w-48' 
+                                type="number"
+                                step="0.01"
+                                min="1" 
+                                value={Amount}
+                                onChange={(e) => setAmount(e.target.value)}
+                                name='donationAmount' 
+                                placeholder='0.00'
+                                required
+                            />
+                    </label>
+                    <textarea 
+                        className='border-2 border-gray-300 p-2 rounded-lg w-full h-48 mt-2' 
+                        type="text" 
+                        name='donationMessage' 
+                        value={Message}
+                        onChange={(e) => setMessage(e.target.value)}
+                        placeholder='Leave a message'
+                        required 
+                        />
+                </div>
+            </div>
+            <div className='text-4xl flex justify-center py-6'>
+                Payment Information
+            </div>
+            <div className=''>
+                    <label>
+                        <div className='text-xl'>Name on Card</div>
+                        <input
+                            className='border-2 border-gray-300 p-2 rounded-lg w-full '
```

Comment:
Maybe make the width a bit smaller? on my screen the w-full is massive and looks waay big compared to the other fields

Reply:
Just made a push, see if that's any better